exporter_test.go

```diff
@@ -1116,6 +1074,116 @@ version = "4.5.6"
 			})
 		})
 
+		when("checking the image manifest", func() {
```

Comment:
Would it make sense to wrap these in a `when("report.toml", ...` like here: https://github.com/buildpacks/lifecycle/blob/d9c7de5395c1ba66682b3e548f8955a11b265c17/exporter_test.go#L1119?

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498/commits/721cf13f78654cd7978864642557e02ce6a9931d